build/js/i18n/ar/index.mjs

```diff
@@ -1,4 +1,7 @@
 import countryTranslations from "./countries.mjs";
 import interfaceTranslations from "./interface.mjs";
 
+export { default as countryTranslations } from "./countries.mjs";
+export { default as interfaceTranslations } from "./interface.mjs";
```

Comment:
Am I right in thinking that lines 4&5 are duplicating the imports of lines 1&2 in this file? Would it also work to change lines 4&5 to just `export { countryTranslations, interfaceTranslations };` to avoid that duplication?

Reply:
It may look that way but if I replace lines 1&2 with 4&5 I get errors on line 7 saying that countryTranslations is undefined.  So it is needed, I tested again this morning.

Reply:
Line 7

```JS
export default { ...countryTranslations, ...interfaceTranslations };
```

Reply:
Sorry I'm away from home today, so I can't check it myself but I actually meant something slightly different. I meant keep lines 1&2, and just replace lines 4&5 with that line I suggested. Do you think that would work?

Reply:
Sorry for the confusion. I understand what you mean now. I just tested it and it works. 

This is what the code looks like
```JS
import countryTranslations from "./countries.mjs";
import interfaceTranslations from "./interface.mjs";

export { countryTranslations, interfaceTranslations };
export default { ...countryTranslations, ...interfaceTranslations };
```

It seems to be working in exactly the same way.

> Sorry I'm away from home today,...

No problem at all, whenever you get the chance. My current translation system is working. I am looking forward to making use of the inbuilt translations.